### ISSUE TITLE
server.ha.disruptionBudget.maxUnavailable: Invalid type. Expected: `null`, given: `integer`

### ISSUE THREAD

**Issue by rule88** [hashicorp/vault-helm#534]
https://github.com/hashicorp/vault-helm/blob/b21b37b07a439e8c0ae6ddd133aca71c719a01ca/values.schema.json#L487

This change prevents upgrading from 0.11.0 to 0.12.0 when using variable `server.ha.disruptionBudget.maxUnavailable`.

**Comment by rule88** [comment]
PR: https://github.com/hashicorp/vault-helm/pull/535

**Comment by rule88** [comment]
Error message:

```
Error: values don't meet the specifications of the schema(s) in the following chart(s):
│ vault:
│ - server.ha.disruptionBudget.maxUnavailable: Invalid type. Expected: null, given: integer
```

**Comment by tvoran** [comment]
Fixed in #535
